shared/middleware/helpers/storage_apps.rb

```diff
@@ -14,7 +14,7 @@ def initialize(storage_id)
   def create(value, ip:, type: nil, published_at: nil)
     timestamp = DateTime.now
     row = {
-      storage_id: @storage_id,
+      storage_id: @storage_id.to_i,
```

Comment:
Presumably, changing it from being broken (returning a hex string) to being broken (returning an integer without a storage_apps row) won't hurt things. The test I was fighting passes locally after changing `SecureRandom.hex` to `SecureRandom.random_number 1_000_000`. Let's see if anything else breaks.